nemo/collections/asr/parts/submodules/multi_head_attention.py

```diff
@@ -256,6 +259,250 @@ def forward(self, query, key, value, mask, pos_emb, cache=None, cache_next=None)
         return out
 
 
+class RelPositionMultiHeadAttentionLocal(RelPositionMultiHeadAttention):
```

Comment:
Add citation for the local attention part

Reply:
And please also add some explanation on the difference between this one and RelPositionMultiHead.